### PR TITLE
Resolve warnings meraki management interface

### PR DESCRIPTION
**Pull request by y0rune**

### CODE REVIEW

File: tests/integration/targets/meraki_management_interface/tasks/main.yml [hunk 1]
```diff
@@ -3,182 +3,199 @@
...
-      var: non_mx_network
+  - name: Debug falsen_mx_network
+    ansible.builtin.debug:
+      var: falsen_mx_network
 
-  - assert:
+  - name: Assert non_mx_network
+    ansible.builtin.assert:
       that:
         - non_mx_network.data is defined
```
Comment (kbreit) [inline tests/integration/targets/meraki_management_interface/tasks/main.yml:92]: This needs to be updated as it's using the old variable name.

**Review by kbreit — CHANGES_REQUESTED** [review]

File: tests/integration/targets/meraki_management_interface/tasks/main.yml (hunk 1 above)
Comment (y0rune) [reply tests/integration/targets/meraki_management_interface/tasks/main.yml:92]: My script changed the `no` to `false`. In that case it changed the `non_mx_network` to `false_mx_network` :D

Comment (y0rune) [reply tests/integration/targets/meraki_management_interface/tasks/main.yml:92]: I pushed it ;)

**Comment by kbreit** [comment]
Thank you for your contribution! Merging.

**Pull request merged**